t/t2200-add-update.sh

```diff
@@ -179,7 +179,10 @@ test_expect_success 'add -u resolves unmerged paths' '
 
 test_expect_success '"add -u non-existent" should fail' '
 	test_must_fail git add -u non-existent &&
-	! (git ls-files | grep "non-existent")
+	! (
+		git ls-files >actual &&
+		grep "non-existent" actual
+	)
```

Comment:
Welcome to the Git project! May I suggest one change that I think will improve your patch even further?

```suggestion
	git ls-files >actual &&
	! grep "non-existent" actual
```

Reply:
@dscho  The code looks cleaner this way and I think it cuts out the redundancy in using the logical not operator. 
Thanks for this and I'll appreciate any more explanations in case I missed out in my understanding.

Reply:
I would suggest this way to go forward:

- accept the change I proposed (there should be a button to commit it)
- pull the changes back into your local branch
- use `git rebase -i` to squash the change into the correct commit (change the third `pick` into `fixup`)
- force-push the branch to your fork
- `/submit` the second iteration of the patch series

I look forward to it!

Reply:
I have made the requested changes. Thanks for your guidance